src/nvim/tui/tui.c

```diff
@@ -350,6 +351,10 @@ static void tui_resize(UI *ui, int width, int height)
   data->scroll_region.left = 0;
   data->scroll_region.right = width - 1;
   data->row = data->col = 0;
+  // try to resize the terminal window
+  char r[16];  // enough for 9999x9999
+  snprintf(r, sizeof(r), "\x1b[8;%d;%dt", height, width);
```

Comment:
Pixels does not look much useful: most of time UI uses character width/height. Depends though how it is supposed to be used: users may prefer pixels (though not all: I do not know how wide some of my terminals are in pixels, yet I normally know how wide they are in characters). Can’t find code where UI resize function receives exactly characters.